test/middleware/logging.test.ts

```diff
@@ -29,25 +35,25 @@ describe('logging', () => {
 
   test('logs requests and responses', () => {
     return request(server).get('/').expect(200).expect(res => {
-      var requestLog = output[0]
-      var responseLog = output[2]
+      const requestLog = output[0]
+      const responseLog = output[2]
 
       // logs id with request and response
       expect(requestLog.id).toBeTruthy()
       expect(responseLog.id).toEqual(requestLog.id)
-      expect(res.headers['x-request-id']).toEqual(requestLog.id)
+      expect(res.header['x-request-id']).toEqual(requestLog.id)
```

Comment:
TypeScript type definition here says we should be using `res.header` instead of `res.headers`. Test is still passing.

Reply:
I'm relying on the incorrect `Response` type here. Will push up another commit.

Reply:
lol nope nevermind, I think this is correct (coming from superagent's TypeScript types)